Resources/Prototypes/_White/Entities/Clothing/Uniforms/jumpsuits.yml

```
@@ -96,3 +96,47 @@
       sprite: _White/Clothing/Uniforms/Jumpsuit/engineer_polar.rsi
     - type: Clothing
       sprite: _White/Clothing/Uniforms/Jumpsuit/engineer_polar.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitMedicalBlue
+  name: Blue Medical Uniform
+  description: NanoTrasen variation of Medical Uniform. It's blue.
+  components:
+    - type: Sprite
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)